### DISCUSSION TITLE
Cannot open on macOS Movaje

### DISCUSSION THREAD

**Discussion by GreaseMonkey88** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#185] [Q&A]
Hi David!
Great app, I have been waiting for this for years! Works fine under Big Sur with my 2018 MBP. I have a Mojave system on an external SSD (for Aperture...), but I can't open the latest release of AlDente 2.1.2 on this system. Here's a screenshot:

<img width="474" alt="Bildschirmfoto 2021-03-11 um 07 54 44" src="https://user-images.githubusercontent.com/47741656/110748486-12365300-8240-11eb-9d60-06c86d5efe2d.png">

Cheers!

EDIT: In the email advertising the latest release I read "Also, this version has been compiled to work with macOS versions **_Mojave_** and newer." But on the Github release page it says Catalina?

**Comment by MatthiasKerbl** [comment]
Hi @GreaseMonkey88,
this was a mistake on our side. AlDente is supported by Catalina and Big Sur, not Mojave.

Matthias

**Reply by yabo-boye** [reply]
hey,

will mojave be supported anytime in the near future?
